README.md

```diff
@@ -77,10 +77,12 @@ $ aegir-test browser
 $ gulp test:browser
 ```
 
-If you want to run tests in a webworker you can also pass `--webworker` as a flag to enable that.
+Browser tests by default run inside the main thread and a webworker thread. If you want to disable or only run your tests in a webworker, you can do it by passing the flag:
 
-If the needed environment variables are set, tests are also run on [Sauce Labs].
-You will need
+- `--webworker=false`
+- `--webworker=only`
```

Comment:
We can keep the `--dom` and `--webworker`, that is the least of the concerns at the moment :) 

When I requested for feedback on it, it was positive and that took me to bring up a PR. What this reminds me is that `aegir` needs desperately a Roadmap with the clear direction of what is missing and what will make it 'done'. Right now it is a core piece of every JS project withing 4 organizations (ipfs, libp2p, multiformats and IPLD) and it is has become more glue code than a tool that is easy to play with.